src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import { useForm } from "react-hook-form";
 import { useFirebase } from "react-redux-firebase";
-import axios from "axios";
```

Comment:
Since we aren't using `axios` here anymore, is there anywhere else we use it still? Can we remove it from our dependencies in `package.json` now? (see https://github.com/sparkletown/internal-sparkle-issues/issues/153)